packages/toolpad-app/src/toolpad/ToolpadShell.tsx

```diff
@@ -178,6 +214,9 @@ function Header({ actions, status }: HeaderProps) {
           }}
         >
           {status}
+          {mode !== null ? (
```

Comment:
can the `mode` ever be `null`? from the code here looks like it can't

Reply:
I agree, the check is not needed. I think the type can remain the same, since (currently) we're not setting it to be "system" anywhere